codespell_lib/data/dictionary.txt

```diff
@@ -3938,6 +3948,7 @@ bianries->binaries
 bianry->binary
 biappicative->biapplicative
 bidimentionnal->bidimensional
+bidings->bindings
```

Comment:
I wonder if this should be in rare too as the plural of biding (I can't remember if the CI requires the left to be a valid word in rare.